src/plugins/intel_gpu/thirdparty/CMakeLists.txt

```diff
@@ -156,25 +156,25 @@ if(ENABLE_ONEDNN_FOR_GPU)
             )
         endif()
 
-        set(LIB_INCLUDE_DIRS "${ONEDNN_INSTALL_DIR}/include" "${CMAKE_CURRENT_SOURCE_DIR}/onednn_gpu/src")
+        set(LIB_INCLUDE_DIRS "${ONEDNN_INSTALL_DIR}/include")
+        set(LIB_INTERNAL_INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}/onednn_gpu/src/"
+                                      "${CMAKE_CURRENT_SOURCE_DIR}/onednn_gpu/src/gpu/intel/jit/ngen/"
+                                      "${CMAKE_CURRENT_SOURCE_DIR}/onednn_gpu/third_party/ngen/"
+                                      "${CMAKE_CURRENT_SOURCE_DIR}/onednn_gpu/third_party/")
         set(LIB_DEFINITIONS ENABLE_ONEDNN_FOR_GPU
                             DNNL_DLL
                             DNNL_DLL_EXPORTS
                             DNNL_ENABLE_CPU_ISA_HINTS
                             DNNL_ENABLE_MAX_CPU_ISA
-                            DNNL_X64=1
-                            NGEN_CPP11
-                            NGEN_NEO_INTERFACE
-                            NGEN_NO_OP_NAMES
```

Comment:
These definitions are supposed to be added here https://github.com/oneapi-src/oneDNN/blob/f039820faaf0536c3dbaa7162db31ee82c0b90b5/src/gpu/intel/jit/ngen/ngen_config.hpp#L24C1-L29C28 Please check if ngen headers are included somewhere w/o including ngen_config.hpp first. If so, maybe we'll need to change that part somehow

Reply:
ngen_config.hpp does not include those definitions yet on rls-v3.8-pc.
https://github.com/oneapi-src/oneDNN/pull/2773 includes it.
I request merge PR#2773 to rls-v3.8-pc to PR author(@rjoursler)

Reply:
fixed by backport PR#2773 to rls-v3.8-pc.